lambda_function.py

```diff
@@ -32,6 +32,10 @@ def lambda_handler(data, context):
                     pass
             elif is_number(first_text) and len(array_text) == 2:
                 text = guess_password(int(first_text))
+            elif 'alvin' in first_text:
+                text = '禿驢'
+            elif 'DROP' or 'DELETE' or 'HELP' or 'help' in first_text:
```

Comment:
`if 'DROP' or 'DELETE' or 'HELP'` always be True lol

Reply:
`if any(text in first_str for text in ['DROP', 'DELETE', 'HELP', 'help']):`